inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/file/FileSourceDTO.java

```diff
@@ -42,9 +42,6 @@ public class FileSourceDTO {
 
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
-    @ApiModelProperty("Agent Cluster tag")
-    private String clusterTag;
-
     @ApiModelProperty("Agent IP address")
     private String ip;
```

Comment:
This field seems can be replaced by `agentIp` in the parent class.

Reply:
This DTO class does not have a parent class. It might be used alone and carry the ip data by whoever needs it.